src/CONST.ts

```diff
@@ -1556,6 +1556,15 @@ const CONST = {
         ];
     },
 
+    // Emails that profile view is prohibited
+    get RESTRICTED_EMAILS() {
+        return [this.EMAIL.NOTIFICATIONS] as string[];
```

Comment:
Please don't typecast. It defeats the purpose of using TypeScript. Instead just add a return type.

```
    get RESTRICTED_EMAILS(): readonly string[] {
        return [this.EMAIL.NOTIFICATIONS];
    },
```